server/src/main/resources/logback.xml

```diff
@@ -39,13 +39,12 @@
         </encoder>
     </appender>
 
-    <logger name="org.dst.server">
-        <level value="info"/>
+    <logger name="org.dst.server" level = "info">
+        <appender-ref ref="CONSOLE"/>
```

Comment:
CONSOLE?

Reply:
This can promise only output the log of server to console.